io-engine/src/bdev/nexus/nexus_bdev_snapshot.rs

```diff
@@ -186,13 +185,13 @@ impl ReplicaSnapshotExecutor {
                 // snapshots.
                 let snapshot_params = SnapshotParams::new(
                     snapshot.entity_id(),
-                    snapshot.parent_id(),
+                    Some(ctx.replica_uuid.clone()),
```

Comment:
why isn't parent_id correct here? because it's the parent id for the "nexus snapshot"?

Reply:
Yes, here parent_id() is nexus id, but We need to update the individual replica_uuid for the each replica snapshot of the nexus.